projects/angular/src/layout/_properties.card.scss

```diff
@@ -6,51 +6,23 @@
-      --clr-card-border-color: var(--clr-color-neutral-300);
-      // АТМ
-      // Leave only `clr-card-box-shadow` and remove `clr-card-box-shadow-color`
-      --clr-card-box-shadow-color: var(--clr-card-border-color);
-      --clr-card-box-shadow: 0 #{variables.$clr_baselineRem_0_125} 0 0 var(--clr-card-box-shadow-color);
-      --clr-card-clickable-border-color: var(--clr-color-action-500);
-      // АТМ
-      // Leave only `clr-card-clickable-box-shadow` and remove `clr-card-clickable-box-shadow-color`
-      --clr-card-clickable-box-shadow-color: var(--clr-card-clickable-border-color);
-      --clr-card-clickable-box-shadow: 0
-        #{variables.$clr_baselineRem_0_125}
-        0
-        0
-        var(--clr-card-clickable-box-shadow-color);
-      --clr-card-text-color: #{variables.$clr-p1-color};
-      --clr-card-text-font-size: #{card-variables.$clr-card-text-font-size};
-      --clr-card-children-space-between: #{variables.$clr_baselineRem_0_5};
-      --clr-card-media-description-margin-left: #{variables.$clr_baselineRem_0_5};
-      --clr-card-media-description-margin-top: #{variables.$clr_baselineRem_0_25};
-      --clr-card-list-group-item-padding: #{variables.$clr_baselineRem_0_5} #{variables.$clr_baselineRem_0_75};
-      --clr-card-header-title-font-size: #{card-variables.$clr-card-title-font-size};
-      --clr-card-content-title-font-size: #{card-variables.$clr-card-title-font-size};
-      --clr-card-header-padding: #{card-variables.$card-children-padding-vertical}
-        #{card-variables.$card-children-padding-horizontal};
-      --clr-card-footer-vertical-padding: #{card-variables.$card-children-padding-vertical}; // ATM: internal property to be deleted
-      --clr-card-footer-padding: #{card-variables.$card-children-padding-vertical}
-        #{card-variables.$card-children-padding-horizontal};
-      --clr-card-block-padding: #{card-variables.$card-children-padding-vertical}
-        #{card-variables.$card-children-padding-horizontal};
-      --clr-card-footer-gap: #{variables.$clr_baselineRem_0_5};
-    }
+  :root [cds-theme] {
+    // Usage: ../layout/_card.clarity.scss
+    --clr-card-bg-color: #{tokens.$cds-alias-object-container-background};
+    --clr-card-divider-color: #{tokens.$cds-alias-object-border-color};
+    --clr-card-title-color: #{tokens.$cds-alias-typography-color-400};
+    --clr-card-title-font-weight: #{tokens.$cds-global-typography-section-font-weight};
+    --clr-card-border-radius: #{tokens.$cds-alias-object-border-radius-100};
+    --clr-card-border-width: #{tokens.$cds-alias-object-border-width-100};
+    --clr-card-border-color: #{tokens.$cds-alias-object-border-color};
+    --clr-card-box-shadow: #{tokens.$cds-alias-object-shadow-300};
+    --clr-card-clickable-border-color: #{tokens.$cds-alias-object-interaction-background-highlight};
+    --clr-card-clickable-box-shadow: #{tokens.$cds-alias-object-shadow-300};
+    --clr-card-header-title-color: var(--clr-card-title-color);
```

Comment:
I had same "transitional" mapping to CDS token for accordions but decided to use cds tokens everywhere, don't know which one we prefer

Reply:
It's just inheriting a color from different custom property while exposing one